forgerock-auth/src/main/java/org/forgerock/android/auth/SSOBroadcastModel.kt

```diff
@@ -0,0 +1,43 @@
+/*
+ *
+ *  * Copyright (c) 2022 ForgeRock. All rights reserved.
+ *  *
+ *  * This software may be modified and distributed under the terms
+ *  * of the MIT license. See the LICENSE file for details.
+ *
+ *
+ */
+
+package org.forgerock.android.auth
+
+import android.content.Context
+import android.content.Intent
+
+
+/**
+ * SSOBroadcastModel to broadcast the SSO SignOut message to single sign on apps
+ */
+
+internal class SSOBroadcastModel(private val context: Context? = InitProvider.getCurrentActivity(),
+                        private val broadcastIntent: Intent = Intent(context?.resources?.getString(R.string.forgerock_sso_logout))) {
+
+    private val broadcastPermission: String? = context?.resources?.getString(R.string.forgerock_sso_permission)
+
+    fun sendBroadcast() {
```

Comment:
Doc?

Reply:
what you think about rename it `sendBroadcast` from  to `sendLogoutBroadcast`? Just in case we decide to add new events in this model in the future

Reply:
Done